### ISSUE TITLE
The aria-describedby attribute is missing the reference element

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3091]
**HC Assessment Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
The "aria-describedby" attribute reference exists in the "Upload a file" button, but the target for the reference does not exist. In the code snippet below, target for reference id "null" in the "aria-describedby" attribute appears to be missing.

<div id="7" role="button" tabindex="0" class="gc-button gc-button--secondary gc-file-input-upload-button" aria-labelledby="file-input-button-text" aria-describedby="7_file_selected desc-7 null" 

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Inspect the HTML source for the "Upload a file" button

Expected result: Ensure the element referenced in the "aria-describedby" attribute value is present within the page and presents a proper label or description.

**Comment by srtalbot** [comment]
Will reassess when we get to file upload this year
